dabudimesh/router.py

```diff
@@ -12,17 +43,20 @@ def add_connection(self, address, socket):
         socket.sendall(bytes(self.address, "utf-8"))
         self.routing_table[address] = socket
 
-    def send(self, address, message):
-        # TODO: raise an error if not found
+    def send(self, address, parameter, command):
         socket = self.routing_table[address]
```

Comment:
What if address not found in routing table? Method send should probably return an error